carsus/io/molecules/molecules.py

```diff
@@ -0,0 +1,225 @@
+import pandas as pd
+from carsus.io.util import read_from_buffer
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+BARKLEM_COLETT_DATA_URL = "https://raw.githubusercontent.com/tardis-sn/carsus-data-molecules-barklem2016/main/data/"
```

Comment:
I would suggest either putting this in a config file (preferred) or having a separate .py file containing this URL.  You may also want to look for mirrors too